CMakeLists.txt

```diff
@@ -79,39 +79,14 @@ endif()
 # Common definitions
 set(EXTRA_DEFINES "__STDC_FORMAT_MACROS")
 
-# Get Python library for linking
-if(APPLE)
-    set(PYTHON_LIB_VAR "LIBRARY")
-else()
-    set(PYTHON_LIB_VAR "LDLIBRARY")
-endif()
 
-execute_process(
-    COMMAND ${Python_EXECUTABLE} -c "import sysconfig; print(sysconfig.get_config_var('${PYTHON_LIB_VAR}') or '')"
-    OUTPUT_VARIABLE PYTHON_LDLIBRARY
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-if(PYTHON_LDLIBRARY)
-    get_filename_component(PYTHON_LIB_LINK ${PYTHON_LDLIBRARY} NAME_WLE)
-    string(REGEX REPLACE "^lib" "" PYTHON_LIB_LINK ${PYTHON_LIB_LINK})
-else()
-    string(REPLACE "." "" PYTHON_VERSION_NODOT ${Python_VERSION_MAJOR}${Python_VERSION_MINOR})
-    set(PYTHON_LIB_LINK "python${PYTHON_VERSION_NODOT}")
-endif()
+# Get Python library for linking
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development.Embed)
```

Comment:
Isn't this using the old interface (https://cmake.org/cmake/help/latest/module/FindPython3.html) where you end up with variables like `Python3_*` instead of `Python_*` (as you get with https://cmake.org/cmake/help/latest/module/FindPython.html)?

Reply:
Changed